src/Compilers/Core/MSBuildTask/Utilities.cs

```diff
@@ -148,20 +150,14 @@ internal static void DeleteNoThrow(string path)
             return new ArgumentException(string.Format(CultureInfo.CurrentCulture, errorString, args));
         }
 
-        internal static string TryGetAssemblyPath(Assembly assembly)
```

Comment:
>TryGetAssemblyPath [](start = 31, length = 18)

is this semantic change necessary? Can it be moved to a separate PR?

Reply:
It's already a separate commit, but I can move it to a prerequisite PR if that's not enough.

Reply:
Sounds like it was not necessary. I'd remove that change from this PR, or request some manual validation, otherwise, the change makes me a bit nervous (not exercised).

Reply:
The reflection APIs return nulls.